INSTALL.md

```diff
@@ -18,26 +19,17 @@ For the complete MARA experiments installation, please refer first to the **ROS2
    - Ubuntu 18: Install ROS 2 Desktop following the official instructions, binaries recommended. [Instructions](https://index.ros.org/doc/ros2/Installation/Linux-Install-Debians/).
 
 ## Dependent tools
-**Note**: We recommend installing **Gazebo 9.0.0** via **ROS Crystal Debian packages** and removing previous gazebo installations to avoid undesired conflicts, e.g. `apt-get remove *gazebo*`. You can also use different versions of the simulator such as Gazebo 10, but you must skip the installation of `ros-crystal-gazebo*` packages and add [gazebo_ros_pkgs](https://github.com/ros-simulation/gazebo_ros_pkgs/tree/crystal) to the `ros2_mara_ws` we are going to build in the [Create a ROS workspace](#create-a-ros-workspace) section.
+**Note**: We recommend installing **Gazebo 9.9.0** via **ROS Crystal Debian packages** and removing previous gazebo installations to avoid undesired conflicts, e.g. `apt-get remove *gazebo*`. You can also use different versions of the simulator such as Gazebo 10, but you must skip the installation of `ros-crystal-gazebo*` packages.
 
 ```sh
 # ROS 2 extra packages
 sudo apt update && sudo apt install -y \
-ros-crystal-action-msgs \
-ros-crystal-message-filters \
-ros-crystal-yaml-cpp-vendor \
-ros-crystal-urdf \
-ros-crystal-rttest \
-ros-crystal-tf2 \
-ros-crystal-tf2-geometry-msgs \
-ros-crystal-rclcpp-action \
 ros-crystal-cv-bridge \
 ros-crystal-control-msgs \
 ros-crystal-image-transport \
 ros-crystal-gazebo-dev \
 ros-crystal-gazebo-msgs \
 ros-crystal-gazebo-plugins \
-ros-crystal-gazebo-ros \
```

Comment:
I think we need this. reset services are located there.

Reply:
Tested on ubuntu:bionic docker and installed ros-crystal-desktop, ros-crystal-gazebo-ros got installed automatically